packages/Webkul/Admin/src/Resources/views/settings/roles/create.blade.php

```diff
@@ -161,7 +161,7 @@ class="primary-button"
                         </option>
                     </x-admin::form.control-group.control>
 
-                    <x-admin::form.control-group.error control-name="permission_type" />
+                    <x-admin::form.control-group.error control-name="permissions" />
                 </x-admin::form.control-group>
 
                 <div v-if="permission_type == 'custom'">
```

Comment:
We can handle the custom validation issue from here `custom type`. Please try to handle here.

Reply:
It is perfect